frontend/test/metabase/scenarios/admin/settings/spinner.cy.spec.js

```diff
@@ -13,10 +13,11 @@ describe("scenarios > admin > spinner", () => {
       cy.findByText("Add Database").should("not.exist");
     });
 
-    it.skip("should not spin forever if it returns an error (metabase#11037)", () => {
+    it("should not spin forever if it returns an error (metabase#11037)", () => {
       cy.visit("/admin/databases/999");
       cy.findAllByText("Databases").should("have.length", 2);
       cy.findByText("Loading...").should("not.exist");
+      cy.findByText("Not found.");
```

Comment:
Good call.
Previously, this test didn't make sense at all, and I'll gladly move it somewhere else. This whole spec is pretty much useless on its own.

We can enable this for now but then move it somewhere else later on. I'm fairly convinced we have some other issue and repro for it that is related to this one.